src/dotnet/APIView/ClientSPA/src/app/_components/shared/related-comments-dialog/related-comments-dialog.component.html

```diff
@@ -17,17 +17,38 @@
       </small>
     </div>
 
-    <!-- Select All Checkbox -->
-    <div class="d-flex align-items-center mb-3 p-2 bg-light rounded">
-      <p-checkbox 
-        [binary]="true"
-        [ngModel]="selectAll"
-        (onChange)="onSelectAllChange($event)"
-        styleClass="me-2">
-      </p-checkbox>
-      <label class="form-check-label fw-semibold">
-        Select All ({{ relatedComments.length }} issues)
-      </label>
+    <!-- Select All Checkbox with inline voting -->
+    <div class="d-flex align-items-center justify-content-between mb-3 p-2 bg-light rounded select-all-section">
+      <div class="d-flex align-items-center">
+        <p-checkbox 
+          [binary]="true"
+          [ngModel]="selectAll"
+          (onChange)="onSelectAllChange($event)"
+          styleClass="me-2">
+        </p-checkbox>
+        <label class="form-check-label fw-semibold mb-0">
+          Select All ({{ relatedComments.length }} issues)
+        </label>
+      </div>
+      
+      <div class="d-flex align-items-center vote-section">
+        <div class="d-flex align-items-center vote-buttons">
+          <button 
+            type="button"
+            class="btn btn-link p-2 vote-btn"
+            title="Upvote selected issues"
+            (click)="toggleBatchVote('up')">
+            <span [ngClass]="hasBatchUpvote() ? 'bi bi-hand-thumbs-up-fill emoji-active fs-5' : 'bi bi-hand-thumbs-up text-muted fs-5'"></span>
+          </button>
+          <button 
+            type="button"
+            class="btn btn-link p-2 vote-btn"
+            title="Downvote selected issues"
+            (click)="toggleBatchVote('down')">
+            <span [ngClass]="hasBatchDownvote() ? 'bi bi-hand-thumbs-down-fill emoji-active fs-5' : 'bi bi-hand-thumbs-down text-muted fs-5'"></span>
+          </button>
```

Comment:
I think these should be moved to be below related commenst and just above the reply box.

Reply:
I might suggest making it flush right so it's located generally where you see it normally, but it looks better to me!

Reply:
<img width="1747" height="757" alt="image" src="https://github.com/user-attachments/assets/306033da-f596-4865-8b68-6c02f995a456" />

changed :)